test/uri/uri_is_absolute_test.cc

```diff
@@ -15,3 +15,8 @@ TEST(URI_is_absolute, urn) {
   const sourcemeta::jsontoolkit::URI uri{"urn:example:schema"};
   EXPECT_TRUE(uri.is_absolute());
 }
+
+// TEST(URI_is_absolute, slash) {
+//   const sourcemeta::jsontoolkit::URI uri{"/foo"};
```

Comment:
https://www.rfc-editor.org/rfc/rfc3986#section-4.2 would let me think that this is absolute: 

> A relative reference that begins with a single slash character is termed an absolute-path reference.

Any thoughts @jviotti ?

Reply:
From what I understand (though I'm not a URI expert :P):

- It is a relative URI
- But it is an absolute _path_

So to answer your question, I would expect `is_absolute()` to be false